org-roam.el

```diff
@@ -1375,6 +1405,41 @@ respectively."
   (setq-local org-roam-current-title (car (org-roam--extract-titles)))
   (add-hook 'after-save-hook #'org-roam--handle-title-change nil t))
 
+(defun org-roam-get-file-id ()
+  "Get the file id of the file in the current buffer."
+  (org-with-point-at (point-min)
+    (when-let ((before-first-heading (= 0 (org-outline-level))))
+      (org-entry-get nil "ID"))))
+
+(defcustom org-roam-sync-update-method 'query
+  "Update method used to sync file information with backlinks.
+
+Valid values are:
+ 'query        Ask before updating link.
+ 'force        Update link without asking.
+ 'nil          Do not update link description unless old description matches link description."
+  :type '(choice
+          (const :tag "Ask before updating link." 'query)
+          (const :tag "Update link without asking." 'force)
+          (const :tag "Do not update link unless old description matches link description." nil))
+  :group 'org-roam)
+
+(defun org-roam-sync-title-to-backlinks ()
```

Comment:
This doesn't seem to be used anywhere.

Reply:
The idea is that by running `org-roam-sync-title-to-backlinks`, I can update backlink discriptions *without* the chaining file name.

Reply:
actually, It is a premature implementation to separate the link between back-links, file name and file title that only works on files whose title doesn't currently match their file name. I will remove it.

Reply:
Need to call this in sync-title-to-filename